packages/eslint-plugin/src/rules/jsx-pragma/__tests__/rule.test.ts

```diff
@@ -173,14 +197,15 @@ import { jsx } from '@compiled/react';
       options: [{ runtime: 'classic' }],
       errors: [
         {
-          message: 'To use the `css` prop you must set the jsx pragma.',
```

Comment:
not sure why `message` was used here at all when the rest of the test cases were using `messageId`

Reply:
Was there a variable it was asserting in the message

Reply:
ahh you're right, i had missed that

Reply:
i'll add some comments clarifying this